content/en/blog/_posts/2022-12-05-Security-Behavior-Analysis/index.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: blog
+title: "All Microservices Are Vulnerable"
+date: 2022-12-05
+slug: security-behavior-analysis
+---
+
+**Author:**
+David Hadas (IBM Research Labs)
+
+As cyber war games continue to intensify in sophistication, organizations deploying cloud services, continue to grow their cyber investments aiming to produce safe and non-vulnerable services. However, the year-by-year growth in cyber investments, does not result in parallel reduction in cyber incidents. Instead, the number of cyber incidents continue to grow annually. Evidently organizations are fighting a battle they are doomed to fail at. No matter how much effort is made to detect and remove cyber weaknesses from deployed services, it seems offenders have the upper hand.
+
+Considering the current spread of offensive tools, sophistication of offensive players, and ever-growing cyber financial gains to offenders, any cyber strategy that relies on constructing a non-vulnerable, weakness-free service in 2023 is clearly too naïve. It seems the only viable strategy is to:
+
+------> **Admit that your services are vulnerable!**
```

Comment:
This is very personal but I would remove the arrow, and shorten it because it looks like a comment or a TODO on a markdown document 😄
```suggestion
**Admit that your services are vulnerable!**
```

Reply:
I agree with @mtardy.

Bold already makes this stick out. I'd credit our readers with the ability to recognize that bold means emphasis. We don't even need the final exclamation mark.

Reply:
Replaced with a Unicode - problem solved